examples/sap_hana_simple_example/main.tf

```diff
@@ -47,4 +47,37 @@ module "gcp_sap_hana" {
   sap_hana_sidadm_uid        = 900
   sap_hana_sapsys_gid        = 900
   address_name               = var.address_name
+  pd_kms_key                 = google_kms_crypto_key.sap_hana_simple.self_link
+}
+
+# Create a KMS key to use as customer managed encryption key for the instance
+# persistent disk. This is completely optional. If you do not need to manage
+# your own keys, just remove this section and remove also the pd_kms_key
+# parameter in the module declaration above.
+resource "google_kms_key_ring" "sap_hana_simple" {
```

Comment:
Could we actually remove the KMS from the simple example (but keep it for HA) or otherwise make an example w/o KMS? I want to test that things continue to work properly without it.

Reply:
Good point. Let's keep the simple example simple, and not use CMEK here. Removed it.